install_config/aggregate_logging.adoc

```diff
@@ -468,81 +469,80 @@ endif::openshift-enterprise[]
 [[scaling-elasticsearch]]
 *Changing the Scale of Elasticsearch*
 
-If you need to scale up the number of Elasticsearch instances your cluster uses
-it is not as simple as changing the number of replicas. This is due to the nature
-of Persistent volumes and how Elasticsearch is configured to store its data and
-recover the cluster.
+If you need to scale up the number of Elasticsearch instances your cluster uses,
+it is not as simple as changing the number of Elasticsearch cluster nodes. This
+is due to the nature of persistent volumes and how Elasticsearch is configured
+to store its data and recover the cluster. Instead, you will need to create a
+deployment configuration for each Elasticsearch cluster node.
 
-Instead, you will need to create a Deployment Configuration for each replica.
+The deployer link:../install_config/imagestreams_templates.html[created
```

Comment:
Slightly vague wording. Does this mean: 

During installation, the deployer creates templates with the Elasticsearch configurations provided to it... ?